webpack.config.js

```diff
@@ -14,6 +14,39 @@ const PATHS = {
     NODE_MODULES: path.resolve(__dirname, 'node_modules'),
 }
 
+const OUTPUTS = [
```

Comment:
not sure if we want to use `bundle.*`?
This would become unclear in local imports
propsal: `bigchaindb-driver.* `